app/src/sharedTest/java/org/oppia/android/app/testing/ImageRegionSelectionInteractionViewTest.kt

```diff
@@ -135,6 +137,26 @@ class ImageRegionSelectionInteractionViewTest {
     }
   }
 
+  @Test
+  // TODO(#1611): Fix ImageRegionSelectionInteractionViewTest
+  @Ignore
+  fun testImageRegionSelectionInteractionView_clickDescRegion3_region3Clicked() {
```

Comment:
Adding screenshot since the test can't run on Robolectric

![image](https://user-images.githubusercontent.com/54740946/130153277-109dc129-50eb-4ad8-9302-a3cb66c6535b.png)

Reply:
Wasn't able to exactly test the content description here

Reply:
What was the exact error that you. are getting here?

Reply:
Re: robolectric I was referring to the issue #1611

Is there a way to retrieve content description of the image region??

Reply:
@FareesHussain Could you explain the issue here ?

Reply:
@aggarwalpulkit596 
The main requirement here is to click on an image region in ImageRegionSelectionInteraction
the only way to exactly retrieve the image region is to add a content description to each image region.
Also included a test case to check the content description

Assigned to review the changes.